backend/src/main/java/com/eyetrade/backend/model/entity/User.java

```diff
@@ -1,30 +1,29 @@
 package com.eyetrade.backend.model.entity;
 
 import com.eyetrade.backend.constants.Role;
-import lombok.Data;
-import org.hibernate.annotations.GenericGenerator;
+import lombok.Getter;
+import lombok.Setter;
 
 import javax.persistence.*;
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotEmpty;
 import javax.validation.constraints.NotNull;
 
-import static com.eyetrade.backend.constants.GeneralConstants.ID_LENGTH;
-
 /**
  * Created by Emir Gökdemir
  * on 12 Eki 2019
  */
-@Data
```

Comment:
@Data is an annotation that includes getter and setter methods of Lombok. In addition that, Lombok knows that it's an entity and it behave as an entity. So we can use @data instead of @getter @setter.